doc/monitoring/when_use_cartridge.rst

```diff
@@ -0,0 +1,221 @@
+.. _when_use_cartridge:
+
+When use Cartridge
+==================
+
+.. _when_use_cartridge-getting_started:
+
+Getting started
+---------------
+
+Если вы используете фреймворк `Cartridge <https://www.tarantool.io/en/doc/latest/book/cartridge/>`__
+для разработки своего приложения, в него уже добавлена перманентная роль metrics, которая подключается по
+умолчанию. Всё, что нужно — указать её в cartridge.cfg и сконфигурировать. Это делается так:
+
+1. Добавляем 'cartridge.roles.metrics' в список ролей cartridge.cfg:
+
+.. code-block:: lua
+
+    local ok, err = cartridge.cfg({
+        roles = {
+            ...
+            'cartridge.roles.metrics',
+            ...
+        },
+    })
+
+2. Добавляем секцию в конфигурацию кластера (либо можно использовать
+`set_export <https://www.tarantool.io/en/doc/latest/book/monitoring/api_reference/#lua-function.metrics.set_export>`__).
+Пример:
+
+.. code-block:: yaml
+
+    metrics:
+    export:
+    - path: '/path_for_json_metrics'
+      format: 'json'
+    - path: '/path_for_prometheus_metrics'
+      format: 'prometheus'
+    - path: '/health'
+      format: 'health'
+
+..  image:: images/role-config.png
+    :align: center
+
+`Learn more <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_dev/#managing-role-specific-data>`_
+about Cartridge configuration.
+
+Здесь мы указываем по каким адресам будут доступны команды и формат ответа. Данные команды теперь доступны
+по адресам:
+
+.. code-block:: shell
+
+    http://url:port/metrics
+    http://url:port/health
+
+Где ``url:port`` — это адрес и порт конкретного экземпляра приложения.
```

Comment:
You need to specify that this is the Cartridge HTTP port, since it could easily be confused with Tarantool listen port